Expensify/expensify-common
# Added ability to give a combobox dropdown a class

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/138718

# Tests

1. Find anywhere that uses a combobox with a dropdown (such as the reports page "submit" button: [link to the code](https://github.com/Expensify/Web-Expensify/blob/4e1bc022194506e9ddcfac78043fdd8bafd1a0f1/site/component/expense/attendeesSelector.jsx#L207) ) and add an attribute in the component called dropDownClasses (`dropDownClasses = "myCoolClass"`)
    <img width="540" alt="Screen Shot 2020-08-12 at 4 42 36 PM" src="https://user-images.githubusercontent.com/49007721/90078797-d7c10380-dcba-11ea-9f78-532ee7f6584c.png">

1. Link JS-Libs to Web-E, open up dev tools and verify that the ul element has the class you added. 

# QA 
Regression testing for combobox dropdowns

1. Go to the Reports page

1. Make or select a Report with an expense

1. Click on the "Submit" button

1. Click in the "To" field.

1. Verify that the dropdown shows and that it no errors pop up on the console.
